### DISCUSSION TITLE
Allow individual owners for different subdomains

### DISCUSSION THREAD

**Discussion by TheBotlyNoob** [vercel/vercel#6378] [Ideas]
I maintain a service for free sub-domains for developers, but they can't use `CNAME` with Vercel, because the base domain is owned by another Vercel account. See https://github.com/is-a-dev/register/issues/489

**Comment by anondev-sudo** [comment]
Yeah, when someone owns test1.domain.com, please allow that another user can add test2.domain.com

Someone may want to use subdomain from friend for project, but now it's not possible :/
